app/views/pages/changelog.html.erb

```diff
@@ -4,9 +4,25 @@
 <div class="space-y-4">
   <h1 class="text-gray-900 text-xl font-medium mb-4">What's New</h1>
   <div class="bg-white shadow-xs border border-alpha-black-25 rounded-xl p-4">
-    <div class="flex justify-center items-center py-20">
-      <p class="text-gray-500">Changelog coming soon...</p>
-    </div>
+    <% @releases_notes.each do |release_notes| %>
+      <div class="flex justify-between gap-4 releases-notes-row">
+        <div class="w-1/3">
+          <div class="px-3 flex items-center gap-3">
+            <div class="text-white shrink-0 w-9 h-9">
+              <%= image_tag release_notes[:avatar], class: "rounded-full w-full h-full object-cover" %>
+            </div>
+          <div>
+            <div class="text-gray-900 font-medium text-sm"><%= release_notes[:name] %></div>
+            <div class="text-gray-500 text-sm"><%= release_notes[:published_at].strftime("%B %d, %Y") %></div>
+          </div>
+        </div>
+        </div>
+        <div class="w-2/3 releases-notes-details">
+          <h1 class="mb-5 text-xl text-gray-900"><%= release_notes[:name] %></h1>
```

Comment:
Small nitpick here, but we should probably change to an `h2` to avoid duplicate H1s on the page.

Reply:
I've changed it to an h2 to avoid duplicate H1s on the page!